src/components/Pagination/styled.tsx

```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components'
 
 export const PaginationPanel = styled.div`
   display: flex;
+  flex: 6;
```

Comment:
Why is it 6 here?
Also this should be another layout setting and should not be written directly in the public component, but where it is used, like `<Pagination style={{ flex: 6 }} />`. Otherwise it will couple the logic of both layouts.